guestbook/views.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+from django.contrib import messages
+from django.shortcuts import render
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django.views.decorators.http import require_http_methods
+
+from .models import Comment
+
+class CommentForm(forms.ModelForm):
+    class Meta:
+        model = Comment
+        fields = ('name', 'email_address', 'comment')
+
+@require_http_methods(('GET', 'POST'))
+def mypage(request):
+    if request.method == 'POST':
+        form = CommentForm(request.POST)
+        if form.is_valid():
+            form.save()
+            messages.info(request, 'Thank you for your comment.')
+
+            return HttpResponseRedirect(reverse('mypage'))
```

Comment:
there's a function called `redirect` in `django.shortcuts` that directly takes a URL name: `return redirect('mypage')`